app/src/components/market/steps/items/funding_and_fee_step.tsx

```diff
@@ -52,22 +52,28 @@ const ErrorStyled = styled.span`
 
 const FundingAndFeeStep = (props: Props) => {
   const context = useConnectedWeb3Context()
+  const { library: provider, account, rawWeb3Context } = context
 
   const { values, addCollateralCustom, handleChange, handleCollateralChange } = props
   const { funding, spread, collateral, collateralsCustom } = values
 
   const calculateCollateralBalance = useMemo(
     () => async (): Promise<BigNumber> => {
-      const collateralService = new ERC20Service(context.library, collateral.address)
-      const collateralBalance = await collateralService.getCollateral(context.account || '')
+      const collateralService = new ERC20Service(
+        provider,
+        rawWeb3Context.connectorName,
+        collateral.address,
+      )
+      const collateralBalance = await collateralService.getCollateral(account || '')
       return collateralBalance
     },
-    [context, collateral],
+    [provider, rawWeb3Context, account, collateral],
   )
 
   const collateralBalance = useAsyncDerivedValue('', new BigNumber(0), calculateCollateralBalance)
 
-  const isFundingGreaterThanBalance = funding.gt(collateralBalance)
+  const isFundingGreaterThanBalance =
+    rawWeb3Context.connectorName !== 'Infura' ? funding.gt(collateralBalance) : false
```

Comment:
We don't have the user address, so we don't have the `collateralBalance` of the user